src/utility/sdk.js

```diff
@@ -32,12 +32,27 @@ const parseServiceMetadata = (response) => {
 };
 
 const fetchServiceMetadata = async (org_id, service_id) => {
+  if(process.env.REACT_APP_SANDBOX) {
+    return {}
+  }
+
   const apiEndpoint = `${APIEndpoints.GET_SERVICE_LIST.endpoint}/org/${org_id}/service/${service_id}/group`;
   return fetch(apiEndpoint).then(res => res.json()).then(parseServiceMetadata);
 };
 
+const generateOptions = (username) => {
+  if(process.env.REACT_APP_SANDBOX) {
```

Comment:
process.env variables would always be casted to Strings.. so this `might` be always true.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309243540)